docs/ADR/2019-05-01-security.md

```diff
@@ -0,0 +1,108 @@
+# [short title of solved problem and solution]
```

Comment:
```suggestion
#  Architecture Security
```

@emettely feel free to change the suggested title

Reply:
Thanks I missed it somehow!